cmd/validate-domains/main.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"io/ioutil"
+	"log"
+	"strings"
+
+	"github.com/asaskevich/govalidator"
+)
+
+var bbListFile = flag.String("file", "../../chaos-bugbounty-list.json", "Chaos bugbounty list json file")
+
+type program struct {
+	Name    string   `json:"name"`
+	URL     string   `json:"url"`
+	Bounty  bool     `json:"bounty"`
+	Swag    bool     `json:"swag"`
+	Domains []string `json:"domains"`
+}
+
+type programs struct {
+	Programs []program `json:"programs"`
+}
+
+func main() {
+	flag.Parse()
+
+	rawJSON, err := ioutil.ReadFile(*bbListFile)
+	if err != nil {
+		log.Fatalf("Failed to read initial JSON file: %v", err)
+	}
+
+	var ps programs
```

Comment:
I'd suggest using `https://github.com/tidwall/gjson`, the whole unmarshaling could be simplified to:

```go
gdata := gjson.ParseBytes(rawJSON)
gdata.Get("programs.#.domains|@flatten").ForEach(func(key, value gjson.Result) bool {
        // domain validation logic here
        domain := value.String()
        ...
	return true
})
```